draft-ietf-rtcweb-jsep.xml

```diff
@@ -3158,7 +3175,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         setRemoteDescription, the following steps MUST be performed:
         <list style="symbols">
 
-          <t>First, the type of the SessionDescription is checked
+          <t>If the description is of type "rollback", follow the
```

Comment:
I think we probably also need to document in this section the PC states in which a setRD(rollback) is valid. It's not totally clear from the rollback section.